junction/tickets/management/commands/fill_data.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Third Party Stuff
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+# Junction Stuff
+from junction.tickets.models import Ticket
+
+
+class Command(BaseCommand):
+    """
+    Read a csv file containing ticket numbers and
+    fill all the details for it.
+    """
+    @transaction.atomic
+    def handle(self, *args, **options):
+
+        if len(args) != 2:
+            raise CommandError('Usage: python manage.py fill_data <in_file> <out_file>')
+
+        in_file, out_file = args
+        ticket_nums = [line.rstrip('\n') for line in open(in_file).readlines()]
+
+        fh = open(out_file, 'w')
+        header = ('Ticket Number', 'Name', 'Email', 'Gender', 'Designation',  'Company', 'City', 'Address')
+        fh.write(','.join(header) + '\n')
+
+        for ticket_num in ticket_nums:
+            ticket = Ticket.objects.get(ticket_no=ticket_num)
```

Comment:
Ticket number may be invalid. Use try catch block and if object doesn't exists, print a message saying `ticket no` not found and move on.